<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Companies</class>
 <widget class="QMainWindow" name="Companies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>500</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pg_info">
     <widget class="QTableWidget" name="table_info_bor">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>40</y>
        <width>330</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_info_lent">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>190</y>
        <width>330</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_borrowed">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>10</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>欠款列表</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_lent">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>借出列表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_trans">
     <widget class="QPushButton" name="btn_submit_trans">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset_trans">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_amount_trans">
         <property name="text">
          <string>金额</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_trans_amt"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_date_trans">
         <property name="text">
          <string>截止日期</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="transfer_date">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="table_trans_lent">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>330</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_lent_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>借出列表</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_trans_bor">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>330</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_borrowed_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>欠款列表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_pur">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>10</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>购买</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>331</width>
        <height>118</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_from_pur">
         <property name="text">
          <string>卖家</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_pur_from"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_amount_pur">
         <property name="text">
          <string>金额</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_pur_amt"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_date_pur">
         <property name="text">
          <string>还款期限</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="purchase_date">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>278</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>90</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_submit_pur">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset_pur">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_finance">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>10</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>向银行融资</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>331</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>30</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_amount_finance">
         <property name="text">
          <string>金额</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_fin_amt"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_date_fin">
         <property name="text">
          <string>期限</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="finance_date">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>268</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>80</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_submit_fin">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset_fin">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_repay">
     <widget class="QPushButton" name="btn_ok_repay">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>220</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请点击以下记录进行选择：</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_repay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>330</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>507</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_info">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>基本信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_transfer">
       <property name="text">
        <string>转移</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_purchase">
       <property name="text">
        <string>购买</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_finance">
       <property name="text">
        <string>融资</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_repay">
       <property name="text">
        <string>还款</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_quit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>40</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lb_total_b">
       <property name="text">
        <string>总欠款金额</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_basic_borr">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lb_total_l">
       <property name="text">
        <string>总借出金额</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_basic_lent">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
